ClojureBridge Stuff & Where it Lives.md

```diff
@@ -6,14 +6,14 @@ All accounts are free unless otherwise noted.
 
 | Account | Who Has the Credentials | Notes |
 |---------|-------------------------|-------|
-| Twitter | Yoko                    | @ClojureBridge |
-| Mailchimp | Bridget, Jen, Jamie    | Email response(?) when people subscribe on clojurebridge.org |
+| Twitter | Board members           | @ClojureBridge |
+| Mailchimp[[*]](#mailchimp) | <strike>Bridget, Jen, Jamie    | Email response(?) when people subscribe on clojurebridge.org</strike> |
```

Comment:
It's not valid to have a `<strike>` span multiple `<td>` elements. Each `<td>` needs its own `<strike>`.

Reply:
Yeah, indeed :) I didn't check that. It is fixed now.